scrapi/base/transformer.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import unicode_literals
+
+import abc
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class BaseTransformer(object):
```

Comment:
I suppose not, though the function of the classes are different enough that it may be easier to understand with them separate. I don't really have a strong opinion one way or the other though.